app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -123,9 +122,6 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
         if($scope.emsCommonModel.metrics_userid != '') {
           $scope.emsCommonModel.metrics_password = $scope.emsCommonModel.metrics_verify = miqService.storedPasswordPlaceholder;
         }
-        if($scope.emsCommonModel.ssh_keypair_userid != '') {
-          $scope.emsCommonModel.ssh_keypair_password = $scope.emsCommonModel.ssh_keypair_verify = miqService.storedPasswordPlaceholder;
-        }
 
```

Comment:
Do not remove this - we actually need this ... so that the `Required` condition is satisfied in Edit mode.
So add this back, except we do not need `ssh_keypair_verify`
```javascript
if($scope.emsCommonModel.ssh_keypair_userid != '') {
  $scope.emsCommonModel.ssh_keypair_password = miqService.storedPasswordPlaceholder;
}
```